setup.py

```diff
@@ -140,6 +140,6 @@ def make_release_tree(self, base_dir, files):
         'build_py': build_py_with_git_version,
         'sdist': sdist_with_git_version
     },
-    #data_files=[('/etc/calyptos/', ['etc/config.yml']),
-    #            ('/usr/share/calyptos/examples/', example_items)],
+    data_files=[('/usr/share/calyptos/etc', ['etc/config.yml']),
```

Comment:
id like this to be /usr/share/calyptos/ rather than /usr/share/calyptos/etc

Therefore, on CentOS 6 the file will live in ```/usr/share/calyptos/config.yml``` and on mac/dev systems ```~/.calyptos/config.yml```